online-shop/src/app/components/presentational/products-details-view/products-details-view.component.html

```diff
@@ -0,0 +1,34 @@
+<html>
+
+<head>
+    <title>Online shop</title>
+</head>
+
+<body>
+    <div class = "header">
+        <div class = "header">
+            <h1>Product: {{product?.name}}</h1>
```

Comment:
If you add a *ngIf="product" above this, you can avoid adding a question mark before the property.